### PR TITLE
Added media folder as location in django nginx conf file

### PR DESCRIPTION
**Pull request by aaboffill**
Include media folder as location in templates/django.conf

### CODE REVIEW

File: templates/django.conf [hunk 1]
```diff
@@ -31,6 +31,9 @@ server {
     location /static/ {
         alias {{ django_path }}/static/;
     }
+    location /media/ {
+        alias {{ django_path }}/media/;
```
Comment (ellmetha) [inline templates/django.conf]: Perhaps you should use the `django_media_path` variable here.

File: templates/django.conf [hunk 2]
```diff
@@ -31,6 +31,9 @@ server {
     location /static/ {
         alias {{ django_path }}/static/;
```
Comment (ellmetha) [inline templates/django.conf]: You could also use the `django_static_path` variable here.

**Review by ellmetha — CHANGES_REQUESTED** [review]

File: templates/django.conf (hunk 1 above)
Comment (aaboffill) [reply templates/django.conf]: @ellmetha done! [here... e01892f](https://github.com/savoirfairelinux/ansible-django/pull/10/commits/e01892fd8bdd5009753875e58e459dd6b07bae40)

**Review by ellmetha — APPROVED** [review]

**Comment by ghost** [comment]
no default values?

**Comment by ghost** [comment]
Oops, sorry, I didn't look properly. I thought these were new variables. nevermind my previous comment.

**Pull request merged**